lib/relation-definition.js

```diff
@@ -1580,9 +1581,10 @@ BelongsTo.prototype.related = function(condOrRefresh, options, cb) {
  * @param {Function} cb Callback of the form function (err, inst)
  * @returns {Promise | Undefined} returns promise if callback is omitted
  */
-BelongsTo.prototype.getAsync = function(options, cb) {
+BelongsTo.prototype.getAsync = // deprecated alias
```

Comment:
Is this method actually used anywhere? I believe `BelongsTo` class is internal, not a part of our public API.